lib/matplotlib/animation.py

```diff
@@ -253,19 +255,19 @@ def __init__(self, fps=5, codec=None, bitrate=None, extra_args=None,
         fps: int
             Framerate for movie.
         codec: string or None, optional
-            The codec to use. If None (the default) the setting in the
-            rcParam `animation.codec` is used.
+            The codec to use. If ``None`` (the default) the ``animation.codec``
+            rcParam is used.
         bitrate: int or None, optional
             The bitrate for the saved movie file, which is one way to control
-            the output file size and quality. The default value is None,
-            which uses the value stored in the rcParam `animation.bitrate`.
-            A value of -1 implies that the bitrate should be determined
-            automatically by the underlying utility.
-        extra_args: list of strings or None
+            the output file size and quality. The default value is ``None``,
+            which uses the ``animation.bitrate`` rcParam.  A value of -1
+            implies that the bitrate should be determined automatically by the
+            underlying utility.
+        extra_args: list of strings or None, optional
             A list of extra string arguments to be passed to the underlying
-            movie utility. The default is None, which passes the additional
-            arguments in the 'animation.extra_args' rcParam.
-        metadata: dict of string:string or None
+            movie utility. The default is ``None``, which passes the additional
+            arguments in the ``animation.extra_args`` rcParam.
+        metadata: Dict[str, str] or None
```

Comment:
I think `dict of string:string` benefits from being changed into `Dict[str,str]` but `... or None` is so prevalent throughout the code base that such a change (if done) should be the object of another PR.